pkg/api/types_test.go

```diff
@@ -873,6 +873,65 @@ func TestRequireRouteTable(t *testing.T) {
 	}
 }
 
+func TestIsPrivateCluster(t *testing.T) {
+	cases := []struct {
+		p        Properties
+		expected bool
+	}{
+		{
+			p: Properties{
+				OrchestratorProfile: &OrchestratorProfile{
+					OrchestratorType: DCOS,
```

Comment:
I'd kinda rather we're testing the `Kubernetes` default case (no `PrivateCluster` object in the api model at all).